### PR TITLE
[athena_0]: HelloWorld pr for users.

### PR DESCRIPTION
**Pull request by martinx**
### What?

* brief: 第一次pr
* author: Martin

### What changes are included in this PR?

包含了一个团队目录、个人目录、个人学习内容。

### Are these changes tested?

测试代码见`main.cpp`，里面编写了一个 **helloworld** 程序

### CODE REVIEW

File: stl/athena/martin/src/01-hello-world/main.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+#include <string>
+using namespace std;
```
Comment (Light-City) [inline stl/athena/martin/src/01-hello-world/main.cpp:3]: You may not use a using-directive to make all names from a namespace available.

see: google style.https://google.github.io/styleguide/cppguide.html#Namespaces

Comment (martinx) [reply stl/athena/martin/src/01-hello-world/main.cpp]: Got it, thanks 👍

Comment (Light-City) [reply stl/athena/martin/src/01-hello-world/main.cpp]: Please submit edits and agree later

**Comment by Light-City** [comment]
Please post a screenshot of the main program running on your computer and let me know that your machine can run C++ programs

**Comment by martinx** [comment]
截屏2023-06-16 17 24 08

**Comment by Light-City** [comment]
The directory structure does not conform to the specification and you should remove .idea dir

**Comment by martinx** [comment]
> The directory structure does not conform to the specification and you should remove .idea dir

截屏2023-06-16 18 44 23

It seems that the wrong pr was replied to. There is no .idea in the submitted file. I am using vs code 😄

**Comment by Light-City** [comment]
> > The directory structure does not conform to the specification and you should remove .idea dir
> 
> 截屏2023-06-16 18 44 23
> It seems that the wrong pr was replied to. There is no .idea in the submitted file. I am using vs code 😄

sorry,

**Comment by Light-City** [comment]
LGTM

**Pull request closed without merging**